x-pack/plugins/infra/public/apps/common_providers.tsx

```diff
@@ -44,15 +45,18 @@ export const CommonInfraProviders: React.FC<{
 export const CoreProviders: React.FC<{
   core: CoreStart;
   plugins: InfraClientStartDeps;
-}> = ({ children, core, plugins }) => {
+  theme$: AppMountParameters['theme$'];
```

Comment:
Curious if there was a reason you went with the indexed type here. The other PRs I looked at (for example https://github.com/elastic/kibana/pull/120003/files) seemed to use `Observable<CoreTheme>`.

Is there any advantage one way or the other?

Reply:
In this case I decided to maintain symmetry with the `setActionHeaderMenu` prop above, which also refers to the type via `AppMountParameters`. So the main advantage is consistency.